snap/snapcraft.yaml

```diff
@@ -152,24 +159,20 @@ parts:
     - libglu1-mesa
     - libgpm2
     - libgtk3-nocsd0
-    - dbus
 
-  snapcraft-desktop-helpers:
-    source: https://github.com/ubuntu/snapcraft-desktop-helpers.git
-    source-subdir: gtk
-    plugin: make
-    make-parameters: ["FLAVOR=gtk3"]
-    build-packages:
-      - build-essential
-      - libgtk-3-dev
+  terminal:
+    plugin: nil
+    stage-packages:
+      - xfce4-terminal
+      - libgdk-pixbuf2.0-bin # GDK pixbuf
+      - libglib2.0-0
     override-build: |
       snapcraftctl build
-      # desktop-launch doesn't handle running without a user session very well, this avoids slow startup here
-      sed --in-place s'/^  needs_update=true/:/' $SNAPCRAFT_PART_INSTALL/bin/desktop-launch
+      update-mime-database $SNAPCRAFT_PART_INSTALL/usr/share/mime
+      rm -f $SNAPCRAFT_PART_INSTALL/usr/share/applications/exo-*
+      rm -f $SNAPCRAFT_PART_INSTALL/usr/share/applications/xfce4-terminal-settings.desktop
```

Comment:
These are _prime_ examples for `prime: - -...`, care to make it so? Won't block. mind you.